index.jsx

```diff
@@ -29,18 +29,23 @@
                 items: []
             };
         },
-        goTo: function(page) {
+        goTo: function(page,e) {
```

Comment:
Could you please match styles? One space after comma, and spaces before and after parentheses.